docs/UserGuide.md

```diff
@@ -190,6 +190,29 @@ Therefore, to avoid unintentionally losing any information while editing tags, w
 * **For advanced users:**
   * You can remove all the tags of an employee with `edit INDEX t/` (see warning above)
 
+#### Removing an employee record : `delete`
+
+1. Get the [index](#glossary) of the employee under the [employee list](#glossary).
+
+<box type="info" seamless>
+If the employee is not found, consider using <a href="">list</a> or any <a href="">find commands</a> to locate the employee in the employee list.
+</box>
+
+2. Type in the following [command](#glossary) in the [command box](#glossary) `delete INDEX` where `INDEX` is the index of the employee in the list currently
+ - For instance, to remove the whole record of the employee indexed 1, type `delete 1` to the command box.
+<box type="warning" seamless>
+Once you delete the record, the records will be no longer available in HRMate and cannot be recovered.
+Therefore, to avoid unintentionally losing any information of the employee, if you just want to modify some information, we recommend using the <a href="#editing-the-name-phone-number-email-address-home-address-or-tags-of-employees--edit">edit</a> command instead to modify the record.
+</box>
+
+3. Press "enter" on your keyboard and you should see the employee removed from the employee list with all leaves related to that employee in the leave list being removed.
+
```

Comment:
Would a screenshot be appropriate here - before and after?

Reply:
may i check if anyone in the process fixing the bug of the leaves removing feature when deleting a person? wait did we identify that as a bug? I think currently when i deleted the person, the leaves related are not deleted. the image can be provided after the bug fixed.